docs-src/go/how-to-schedule-a-workflow-in-go.md

```diff
@@ -0,0 +1,233 @@
+---
+id: how-to-schedule-a-workflow-in-go
+title: How to Schedule a Workflow in Go 
+sidebar_label: Schedule
+description: Set a Schedule in Go for scheduling Workflows to run at specific times.
+tags:
+  - go
+  - developer-guide
+  - how-to
+---
+
+Scheduling [Workflows](/concepts/what-is-a-workflow) is crucial to automation.
+By scheduling [Workflow Executions](/concepts/what-is-a-workflow-execution), you can reduce manual intervention and ensure timely execution of business processes.
+
+Temporal allows you to execute multiple Schedule operations, either through code or with the [CLI tool](/concepts/what-is-the-temporal-cli).
+Read on to find out how to create and edit Schedules.
+
+:::note
+Be sure to [enable Schedules in your environment](/concepts/what-is-a-schedule#limitations) before creating any Schedules.
+:::
+
+## Create
+
+Schedules are created with the `create` action.
+For each new Schedule, tbe Temporal Server generates a unique Schedule ID.
+
+To create a Schedule in Go, use `ScheduleClient().Create()` on the [Client](/concepts/what-is-the-temporal-client).
+Schedules must be initialized with a Schedule ID, [Spec](/concepts/what-is-a-schedule#spec), and [Action](/concepts/what-is-a-schedule#action) to perform.
+Enter these values in `client.Schedule.Options{}`.
+
+<!--SNIPSTART samples-go-schedule-create-delete {"selectedLines": ["26-38"]}-->
+
+[schedule/starter/main.go](https://github.com/temporalio/samples-go/blob/master/schedule/starter/main.go)
+
+```go
+// This schedule ID can be user business logic identifier as well.
+scheduleID := "schedule_" + uuid.New()
+workflowID := "schedule_workflow_" + uuid.New()
+// Create the schedule, start with no spec so the schedule will not run.
+scheduleHandle, err := c.ScheduleClient().Create(ctx, client.ScheduleOptions{
+	ID:   scheduleID,
+	Spec: client.ScheduleSpec{},
+	Action: &client.ScheduleWorkflowAction{
+		ID:        workflowID,
+		Workflow:  schedule.SampleScheduleWorkflow,
+		TaskQueue: "schedule",
+	},
+})
+if err != nil {
+	log.Fatalln("Unable to create schedule", err)
+}
+```
+
+<!--SNIPEND-->
+
+## Backfill
+
+Backfilling a Schedule executes [Workflow Tasks](/concepts/what-is-a-workflow-task) ahead of the Schedule's specified time range.
+This is useful when you need to execute a missed or delayed Action, or if you want to test the Workflow ahead of time.
+
+To backfill a Schedule in Go, use `Backfill()` on `scheduleHandle`.
```

Comment:
nit: `scheduleHandle ` -> `ScheduleHandle ` because `ScheduleHandle` is the actual type